demo/client/backend/src/server.js

```diff
@@ -0,0 +1,48 @@
+/*
+* Copyright IBM Corp All Rights Reserved
+*
+* SPDX-License-Identifier: Apache-2.0
+*/
+
+'use strict';
+
+const fabricClient = require('./fabricClient');
+const ccRoute = require('./ccRoute');
+const apiRoute = require('./apiRoute');
+const clockauctionRoute = require('./clockauctionRoute');
+
+////////////////////////////  Global constants  /////////////////
+//  errorcodes
+const SUCCESS = 0;
+const FAILURE  = 1;
+
+//  SJ: install express
+var express = require('express');
+var app = express();
+app.use(express.json());
+app.use(function (req, res, next) {
+    res.header("Access-Control-Allow-Origin", "*");
+    res.header("Access-Control-Allow-Headers", "Origin, X-Requested-With, Content-Type, Accept");
+    res.header("Access-Control-Allow-Methods", "PUT, POST, GET, DELETE, OPTIONS");
```

Comment:
we dont use PUT and DELETE, OPTIONS, right? Even though I know that the I sends indeed some OPTIONS, but I dont think this is needed. We could be more restrictive here